docs/source/deploypeer/peerchecklist.md

```diff
@@ -0,0 +1,406 @@
+
+    # Statically defines peer to be an organization "leader",
+    # where this means that current peer will maintain connection
+    # with ordering service and disseminate block across peers in
+    # its own organization. Multiple peers or all peers in an organization
+    # may be configured as org leaders, so that they all pull
+    # blocks directly from ordering service.
+    orgLeader: true
+
+    # Gossip state transfer related configuration
+    state:
+        # indicates whenever state transfer is enabled or not
+        # default value is true, i.e. state transfer is active
+        # and takes care to sync up missing blocks allowing
+        # lagging peer to catch up to speed with rest network
+        enabled: false
+
+    pvtData:
+      implicitCollectionDisseminationPolicy:
+          # requiredPeerCount defines the minimum number of eligible peers to which the peer must successfully
+          # disseminate private data for its own implicit collection during endorsement. Default value is 0.
+          requiredPeerCount: 0
+
+          # maxPeerCount defines the maximum number of eligible peers to which the peer will attempt to
+          # disseminate private data for its own implicit collection during endorsement. Default value is 1.
+          maxPeerCount: 1
+```
+
+Peers leverage the Gossip data dissemination protocol to broadcast ledger and channel data in a scalable fashion. Gossip messaging is continuous, and each peer on a channel is constantly receiving current and consistent ledger data from multiple peers. While there are many Gossip parameters that can be customized, there are three groups of settings you need to pay attention to at a minimum:
+
+* **Endpoints** Gossip is required for block dissemination (not recommended other than for private data transactions) and service discovery (recommended). To use any of these features, you must configure the gossip `bootstrap`, `endpoint`, and `externalEndpoint` parameters in addition to **setting at least one anchor peer** in the peer’s channel configuration.
+
+  - **`bootstrap`**: (Default value should be overridden.) Provide the list of other peer [addresses](#address) in this organization to discover.
+
+  - **`endpoint`**: (Default value should be overridden.) Specify the address that other peers _in this organization_ should use to connect to this peer. For example, `peer0.org1.example.com:7051`.
+
+  - **`externalEndpoint:`** (Default value should be overridden.) Specify the address that peers in _other organizations_ should use to connect to this peer, for example, `peer0.org1.example.com:7051`.
+
+* **Block dissemination** As of Fabric v2.2, in order to reduce network traffic, it is recommended that peers get their blocks from non-private data transactions from the ordering service instead of from other peers in their organization. The combination of the `useLeaderElection:`, `orgLeader:`, and `state.enabled` parameters in this section control this behavior.
```

Comment:
```suggestion
* **Block dissemination**  In order to reduce network traffic, it is recommended that peers get their blocks from the ordering service instead of from other peers in their organization (the default configuration starting in Fabric v2.2). The combination of the `useLeaderElection:`, `orgLeader:`, and `state.enabled` parameters in this section ensures that peers will pull blocks from the ordering service.
```